README.md

```diff
@@ -22,19 +22,32 @@ prawn_document do |pdf|
 end
 ```
 
-No need to call `pdf.render`, it is called by `prawn_document`
+No need to call `pdf.render`, it is called by `prawn_document`.
 
-If you want to customize the name of the file should a user try to save it, you can specify the filename in your action:
+You can customize the name of the generated PDF and the file's delivery format by adding a `prawn_options` method to your controller:
+
+```ruby
+def prawn_options
+  {
+    filename: "my-new-document.pdf",
+    disposition: "attachment"
+  }
+end
```

Comment:
Im not sure this approach is the silver bullet. With this implementation, we cannot set two different filenames or different dispositions for controllers that contain more than one PDF action. Can we not set these options from within the `prawn_document` method?

Reply:
We absolutely can - I'd honestly prefer that approach to doing this via a helper in the Renderer, but I didn't want to break functionality for anyone rendering without using `prawn_document`. 

Would you prefer these changes as an extension of the `prawn_document` method's options exclusively, or would you prefer both this option _and_ something like `prawn_document(filename: "derp.pdf")`?

Reply:
Im not sure if anyone is using it without `prawn_document`. But yes lets prefer `prawn_document` changes only instead of the `prawn_options` method

Reply:
And yes, `filename` should also be a supported argument.